src/main/scala/tile/Core.scala

```diff
@@ -30,6 +30,7 @@ trait CoreParams {
   val useCryptoSM: Boolean
   val useRVE: Boolean
   val useConditionalZero: Boolean
+  val useMulDivWithoutDiv: Boolean
```

Comment:
context: We had some prior private communication.

After seeing the code things become more clear to me, It seems that `useMulDivWithoutDiv` can be merged into `MulDiv` by setting `divUnroll = 0` instead of a standalone option?

I favor this simple way. And for the exposed interface to the end user, he/she only needs to use some config like `WithoutDiv` and that config will set `divUnroll = 0`

But this is tricky as `divUnroll == 0` has two purpose for developer now

1. Zmmul, namely turning off div.
2. Alternative divider impl suggested in #1228

Reply:
After discussion with Jerry, it is concluded that adding another option inside `MulDivParams` like `divUnroll: Option[Int]` or (`divUnroll: Int` and `divEnable: Boolean = true`) should be used, instead of another boolean in `CoreParams`. The later way is prefered as it wont break compatibility.